src/components/PeopleFilters.tsx

```diff
@@ -33,63 +73,76 @@ export const PeopleFilters = () => {
       <div className="panel-block">
         <div className="level is-flex-grow-1 is-mobile" data-cy="CenturyFilter">
           <div className="level-left">
-            <a
+            <SearchLink
               data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=16"
+              className={classNames('button mr-1', {
+                'is-info': centuries.includes('16'),
+              })}
+              params={toggleCenturies('16')}
             >
               16
-            </a>
-
-            <a
+            </SearchLink>
+            <SearchLink
               data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=17"
+              className={classNames('button mr-1', {
+                'is-info': centuries.includes('17'),
+              })}
+              params={toggleCenturies('17')}
             >
               17
-            </a>
-
-            <a
+            </SearchLink>
+            <SearchLink
               data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=18"
+              className={classNames('button mr-1', {
+                'is-info': centuries.includes('18'),
+              })}
+              params={toggleCenturies('18')}
```

Comment:
The 'toggleCenturies' function is being called immediately here, which means it will execute during render and not when the button is clicked. You should pass a function reference instead to ensure it executes on click.